Source/WebCore/css/StyleColor.h

```diff
@@ -60,8 +81,38 @@ struct StyleColor {
         Current = 1 << 1,
         System = 1 << 2,
     };
+
     // https://drafts.csswg.org/css-color-4/#typedef-color
     static bool isColorKeyword(CSSValueID, OptionSet<CSSColorType> = { CSSColorType::Absolute, CSSColorType::Current, CSSColorType::System });
+
+    StyleColor();
+    StyleColor(const Color&);
+    StyleColor(const ColorKind&);
```

Comment:
ColorKind should be an internal detail, and this constructor private . You could have for example

static StyleColor StyleColor::currentColor()  

for constructing the values instead.